### DISCUSSION TITLE
Retrieve Domain deleted from Team

### DISCUSSION THREAD

**Discussion by calebba** [vercel/community#7056] [Help] [answered]
### Summary

I want to retrieve a domain I deleted from the team "bawacaleb14gmailcom's projects". 

### Steps to Reproduce

1. Login
2. Settings
3. Domains
4. Click on the 3 dots and deleted

**Comment by versecafe** [comment]
Did you purchase the domain through a third party registrar or through Vercel directly?

**Reply by calebba** [reply]
It was purchased through Vercel directly

**Reply by versecafe** [reply]
you'll want to reach out to the Vercel support team directly than and make sure you can bring proof of account ownership or purchase and they should be able to help you out.

**Reply by calebba** [reply]
I am using the hobby for now so I want to verify if retrieving the domain is possible before I pay for the pro

**Reply by amyegan** [reply, marked as answer]
**You do not need to pay for Pro to recover a purchased domain.** In many cases, you can recover it yourself by [adding it to the domains list](https://vercel.com/docs/projects/domains/add-a-domain#add-and-configure-domain). You may need to add it [to your personal account](https://vercel.com/account/domains) before [moving the domain to new team](https://vercel.com/docs/projects/domains/transfer-your-domain).

If that doesn't work, you can [submit a Domain Renewals case](https://vercel.com/help) to the Customer Success team who can work with you to verify domain ownership and move it back to your account.

**Reply by calebba** [reply]
I had the domain in my account and it expired. I wanted to renew and mistakenly deleted it. Now when I try to add it again, it appears to not be available.

**Reply by amyegan** [reply]
In that case, the best thing is to [submit a Domain Renewals case](https://vercel.com/help) to the Customer Success team. They're able to access the billing info and domain purchase history needed to help with domain renewal.
